index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Disappear</title>
+
+<style>
+
+    .block {
+        display: flex;
+        justify-content: space-around;
+        padding: 100px;
+    }
+
+    .one-img {
+        width: 400px;
+        height: 300px;
+        opacity: 0;
+        transition: 1s;
```

Comment:
Додати б свойство...
```suggestion
        transition: opacity 1s;
```